app/controllers/home_controller.rb

```diff
@@ -1,5 +1,7 @@
 class HomeController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
This looks like it's reapeated in a bunch of controllers. Should we move it up to the parent controller, `ApplicationController`?